fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBStoreTimer.java

```diff
@@ -570,6 +570,10 @@ public String logKey() {
         ONLINE_INDEX_BUILDER_RECORDS_SCANNED("number of records scanned by online index build", false),
         /** The number of records indexed by {@link OnlineIndexer}. */
         ONLINE_INDEX_BUILDER_RECORDS_INDEXED("number of records indexed by online index build", false),
+        /** The number of indexer range termination by scan limit {@link OnlineIndexer}. */
```

Comment:
Maybe you can explain more about "range termination" here.

Reply:
Yes, this is very unclear. How about: 

```suggestion
        /** The indexer's single transaction range limit - by number of records scanned {@link OnlineIndexer}. */
        ONLINE_INDEX_BUILDER_RANGE_BY_COUNT("number of indexer iterations terminated by scan limit", false),
        /** The indexer's single transaction range limit - by transaction's write size  {@link OnlineIndexer}. */
        ONLINE_INDEX_BUILDER_RANGE_BY_SIZE("number of indexer iterations terminated by write limit", false),
```

Reply:
I think that the current comment seems to suggest that the timer's will contain, like, the values of the limits themselves rather than the number of times a transaction is committed because it hits each limit (which I think is what it's actually recording, right?).

Reply:
I've changed that according to your suggestion below. Please review the change.